libs/math/include/math/vec4.h

```diff
@@ -84,10 +84,16 @@ class MATH_EMPTY_BASES TVec4 :
 
     // default constructor
     constexpr TVec4() = default;
+#if 0
+    constexpr TVec4(const TVec4&) = default;
+
+    template <typename... A> 
+    constexpr TVec4(A... as) : v{as...} {} 
+#endif
 
     // handles implicit conversion to a tvec4. must not be explicit.
     template<typename A, typename = enable_if_arithmetic_t<A>>
-    constexpr TVec4(A v) : v{ T(v), T(v), T(v), T(v) } {}
+    constexpr explicit TVec4(A v) : v{ T(v), T(v), T(v), T(v) } {}
```

Comment:
no this shouldn't be explicit -- that would explain why the implicit conversions dont work above.
we really want implicit conversions to vectors

Reply:
issue is that float can convert implicitely to both tvec3 and tvec4 it causes an ambiguity.

Reply:
filament still has build errors if I remove this `#if 0`, even with #1695

Reply:
Are there still some errors with `explicit`? What are they?

Reply:
I believe the `explicit` won't be needed with my last change